build.gradle

```diff
@@ -72,6 +72,7 @@ allprojects {
 
     repositories {
         // Do not use `mavenCentral()`, it prevents Dependabot from working properly
+        // mavenCentral() // TODO Where to get com.facebook.react:react-android besides from mavenCentral?
```

Comment:
@bmarty , What repo should I add to get the `com.facebook.react:react-android` dependency?

Reply:
I think you add to add the group `com.facebook.react` in the list here: https://github.com/element-hq/element-android/blob/develop/dependencies_groups.gradle#L58

Note: I am doing an upgrade to Jitsi 10.2.0 so maybe not useful to spend more time on this.